profiles/2016_04_01_a549_48hr_batch1.dvc

```diff
@@ -0,0 +1,5 @@
+outs:
+- md5: 856a0e07b38c2cdeae7822b80d8f8ee9.dir
+  size: 1285922999
+  nfiles: 816
```

Comment:
Wow nice, single file for the directory

Reply:
@gwaygenomics I found it odd that 1620 gz files were removed but apparently only 816 were added. Oh wait, that's because Batch2 has the remaining 804 
https://github.com/gwaygenomics/lincs-cell-painting/blob/38fb9f65279c734c3ce37205a358708bdca55c51/profiles/2017_12_05_Batch2.dvc#L4
All set